src/CommandManager.ts

```diff
@@ -45,7 +45,7 @@ export class CommandManager {
 	 * @param {boolean} returnAlias true to also return which alias of the command was used
 	 * @return {Command}
 	 */
-	find(search: string, returnAlias: boolean) {
+	find(search: string, returnAlias?: boolean) {
```

Comment:
Good catch here. I feel like there may be other actually-optional arguments lurking around the codebase.